### ISSUE TITLE
[BUG]Search in downloads does not work

### ISSUE THREAD

**Issue by The-Droidster** [LivingWithHippos/unchained-android#190]
Hello,

Searching in the Downloads section does not return any results although there are multiple files with filenames containing the search term. It seems that only the last few links are being indexed.

A screen recording to demonstrate the issue:
https://photos.app.goo.gl/9TgANkU8DoDnW3mb8

Let me know if I can provide any logs.

**Comment by LivingWithHippos** [comment]
That's strange, it's not happening to me. Do you have a lot of items in your downloads list?

**Comment by The-Droidster** [comment]
> That's strange, it's not happening to me. Do you have a lot of items in your downloads list?

Yes, it's a pretty big list, but then that's where the search function is actually useful.

**Comment by redx113** [comment]
I can confirm the same bug. The search when its large talking over 1000, the search simply doesnt work. Probably cause when its past 50, on realdebrid site it considers that a new page, and when going down past 50 youll see as scroll down it will repeat stuff as it goes to next page of stuff

**Comment by boognish-rising** [comment]
I was just about to report this as well. I hadn't surmised that it was related to the number of items contained within Downloads, but it happens that there are probably upwards of 1,000 or better atm. All I planned to report was that search functionality seems broken. I literally had a download containing the search query on the screen and typed it verbatim and it pulled zero results. This, of course, being after the search function had failed who knows how many times and I had switched gears to discover the extent to which it fails - it appears the answer is, "total' / "absolute." At least once the size of the index reaches a certain point, as previously indicated.

**Comment by LivingWithHippos** [comment]
Real Debrid does not support searching, what the search does is filtering the results pulled from the download/torrent endpoints. If 50 results have been pulled the search will be done on those 50 etc.
You can change in the settings the number of results pulled to a bigger number as a workaround
